ios/Samples/MaasSample/LoginSample/MaasConfiguration.swift

```diff
@@ -0,0 +1,46 @@
+import MaasCore
+import Combine
+
+class MaasConfiguration: ApiConfig {
+
+    private init() {}
+
+    static let shared = MaasConfiguration()
+    static var config: CoreBinary.ApiConfiguration { shared.config }
+
+    @UserDefault(key: "accessToken", defaultValue: nil)
+    static var accessToken: String?
+
+    @UserDefault(key: "temporaryAccessToken", defaultValue: nil)
+    static var temporaryAccessToken: String?
+
+    var baseUrl: String { "https://api-trafi.bedrock.dev.trafi.com/" }
+    var apiKey: String { "OwJwMJr8x11WfgqJlTmZJ3cpXHBPIdob3TNvTgmk" }
```

Comment:
We need to find a way to not include these credentials in the repository.

Reply:
We could leave these empty.

Reply:
In the Android sample apps I specify these via local environment variables that are used to generate resources during build time. What is the best way to pass local parameters to the Xcode build system that can be referenced in the source code?

Reply:
We could define these in *.plist or in project environment variables. At the moment i'll move these in to environment variables.

Reply:
cc @justasm https://github.com/trafi/maas-components/pull/114/commits/c9a1efd44c874fa8052f9715ec7843b31545f29f